### ISSUE TITLE
Duplicate mail notification, when moving mail on thunderbird desktop

### ISSUE THREAD

**Issue by ChristopherWalz** [thunderbird/thunderbird-android#7768]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.802

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Samsung A52s

### Steps to reproduce

1. Have thunderbird opened on desktop
2. Have a thunderbird filter rule, which moves a message from the inbox to another folder
3. Receive a message which triggers the filter rule and gets moved to another folder
4. K-9 will receive 2 notifications about the same message

-----------------------------

IMAP account on my own website (email provider is all-inkl)

### Expected behavior

Receive 1 notification

### Actual behavior

Receive 2 notifications

**Comment by cketti** [comment]
Client side filters can't avoid the race condition where more than one client could see a new message before one of them is able to move the message to a different folder.

K-9 Mail should remove the notification for the message in the inbox once it has been moved to a different folder. However, if you enabled notifications for the folder the message is moved into, you will still get two notifications (with the first one being removed once the app gets notified that the message has been removed from the inbox).

Your options are:
- Use server-side filtering. That way messages are moved before any client is able to see it in the inbox.
- Disable notifications for the folder the message is moved into.

**Comment by ChristopherWalz** [comment]
Correct. The first notification should be removes, so we would only end up with one notification?

**Comment by cketti** [comment]
So you're saying the first notification is not removed for you? Can you please record a debug log while receiving a message that is moved to a different folder by a filter?

**Comment by ChristopherWalz** [comment]
It is not removed. How to record a debug log? I will try tomorrow.

**Comment by cketti** [comment]
https://github.com/thunderbird/thunderbird-android/wiki/LoggingErrors

**Comment by cketti** [comment]
The wiki page contains instructions on how to export the log from within the app.

> If the app isn't crashing during startup, you can export the debug log from inside the app. Go to Settings → General settings → Debugging, then select Export logs from the menu. You'll be prompted for a location to save the log to.

**Comment by ChristopherWalz** [comment]
Ok, here is the log file. I had to remove some lines in the beginning (hopefully nothing important) and anonymize everything else. I hope you will still be able to find the problem.

**Mail info**
From: Foo Bar <from@example.com>
To: to@example.com
subject: test_mail_subject

App options:
- disabled unified folder
- only manual email retrieve
- push directories: all

[k9_logs.txt](https://github.com/thunderbird/thunderbird-android/files/15092891/k9_logs.txt)
